recbole/utils/utils.py

```diff
@@ -48,8 +48,7 @@ def ensure_dir(dir_path):
         dir_path (str): directory path
 
     """
-    if not os.path.exists(dir_path):
-        os.makedirs(dir_path)
+    os.makedirs(dir_path, exist_ok=True)
```

Comment:
In DDP, when multiple processes jointly query whether a file/dir exists, the previous method will report an error because there is no thread lock.